pinot-core/src/main/java/org/apache/pinot/core/transport/QueryRouter.java

```diff
@@ -92,6 +88,16 @@ public AsyncQueryResponse submitQuery(long requestId, String rawTableName,
       @Nullable Map<ServerInstance, ServerRouteInfo> offlineRoutingTable,
       @Nullable BrokerRequest realtimeBrokerRequest,
       @Nullable Map<ServerInstance, ServerRouteInfo> realtimeRoutingTable, long timeoutMs) {
+    TableRouteInfo tableRouteInfo = new ImplicitHybridTableRouteInfo(offlineBrokerRequest, realtimeBrokerRequest,
```

Comment:
Is this method only for testing purpose? If so, let's refactor the test to use the one used in production code

Reply:
Also in `PinotServerDataFetcher.scala`. You dont see a diff because I kept this function to avoid changes in that file.